### PR TITLE
feat/Build Homepage layout

### LINKED ISSUE #56: Build Homepage layout

### PR DESCRIPTION
**Pull request by sumeetrohra**
close #56

## Types of changes

- [x] Add tags to auto close the corresponding issue.
- [x] I followed branch and commit naming convention.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: client/src/components/BlogCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './BlogCard.module.css';
+
+export default function BlogCard({ data }) {
+  const {
+    category, title, content, author, image,
+  } = data;
+  console.log(data);
```
Comment (avinash360) [inline client/src/components/BlogCard/index.jsx]: remove console.log

File: client/src/components/BlogList/index.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  const conditionallyRenderBlogs = () => {
+    if (blogs.length === 0) {
+      return <LoaderCentered />;
+    }
+    return blogs.map((blog) => {
+      const refactoredData = {
+        ...blog,
+        content: blog.content || 'Some description...',
+        image: blog.image || 'https://source.unsplash.com/aJTiW00qqtI',
```
Comment (avinash360) [inline client/src/components/BlogList/index.jsx]: create a config file to get common data for each environment, do not hard code image path

Comment (sumeetrohra) [reply client/src/components/BlogList/index.jsx]: we'll have same data for each environment, so no need for a config file, will create a new file for constants though

Comment (avinash360) [reply client/src/components/BlogList/index.jsx]: even though we have same data for each environment, urls can change, so its better to have a config file, that takes environment variables from netlify.

**Pull request merged**